### ISSUE TITLE
Support a default .travis.yml for subpackages

### ISSUE THREAD

**Issue by jakemac53** [dart-lang/mono_repo#3]
Most of the packages in https://github.com/dart-lang/build want the exact same travis config (actually, all of them I think), but today they all need their own .travis.yml file.

It would be nice if I could provide like a default.travis.yml in the root, and it would apply that to any package which didn't have one.

**Comment by matanlurey** [comment]
So, @jakemac53, is this done, sorta?

**Comment by jakemac53** [comment]
not really, I changed the title though now that we are using .mono_repo.yml files

**Comment by donny-dont** [comment]
@jakemac53 what about a root `.mono_repo` which would have files you could sync over? Could maybe even use some of the package info or something from dependent pubspecs?

**Comment by kevmoo** [comment]
Maybe `mono_pkg.shared.yml`?

**Comment by kevmoo** [comment]
...or a key within mono_repo.yml?
